docs/other_vscode.md

```diff
@@ -77,11 +77,11 @@ Now, we will set up the MSYS2 window to show up in VSCode as the integrated term
     }
     ```
 
-   If there are settings here already, then just add everything between the first and last curly brackets.  
+   If there are settings here already, then just add everything between the first and last curly brackets and separate the existing settings with a comma from the newly added ones.
 
 ?> If you installed MSYS2 to a different folder, then you'll need to change the path for `terminal.integrated.shell.windows` to the correct path for your system. 
 
-4. Hit Ctrl-` (grave) to bring up the terminal.  
+4. Hit Ctrl-` (grave) (for US layouts) to bring up the terminal or go to <kbd>Terminal</kbd> > <kbd>New Terminal</kbd>
```

Comment:
``Ctrl-` `` is true for other host layouts, for instance UK.

Reply:
hmm ok, didn‘t think of that. Gonna try and find a solution to make this more helpful without being to specific about the layout. I even added a backtick to my keyboard to try the shortcut and saw that it did not work with de_CH so I wanted to clear that up and though its only for US layout. Gonna come back to this and change it later.